CHANGELOG.md

```diff
@@ -4,6 +4,10 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
+## Upcoming release
+- Export ModelModalComponent for direct use outside of this library
+- Rework ModelModalComponent filtering and hiding behavior to avoid triggering updates when parent chart has been removed from DOM.
```

Comment:
extreme nitpick but this bullet ends with a period and the others don't

Reply:
Thanks. I like the consistency as well.